api/clever/__init__.py

```diff
@@ -254,15 +253,18 @@ def remote_patron_lookup(self, token):
             with the data listed above.
 
         """
-        bearer_headers = {'Authorization': 'Bearer %s' % token}
+        bearer_headers = {
+            "accept": "application/json",
```

Comment:
Just personal preference but I think this looks better capitalized as `Accept`, but it is case-insensitive so it'll work either way.